common/Reducers.jsm

```diff
@@ -596,7 +597,7 @@ function DiscoveryStream(prevState = INITIAL_STATE.DiscoveryStream, action) {
       return {
         ...prevState,
         spocs: {
-          ...INITIAL_STATE.DiscoveryStream.spocs,
+          ...prevState.spocs,
```

Comment:
why this change? won't prevState got reset to INITIAL_STATE.DiscoveryStream when spoc end point change